src/cls/IPM/Main.cls

```diff
@@ -250,6 +250,9 @@ load -dev -verbose C:\module\root\path\module-0.0.1.tgz
 load https://github.com/user/repository.git
 load https://github.com/user/repository.git -branch feature-1
 </example>
+<example description="Loads the module described in C:\module\root\path\module.xml without installing python dependencies">
+	load -DBypassPyDeps=1 C:\module\root\path\
```

Comment:
Instead of using a -D modifer which doesn't get documented in a method's regular commands, can you explicitly add the modifier to the load and install commands?
Long term, would be nice to get rid of -D modifiers and only deal with those that are registered. Custom lifecycle/processor classes can register similar to how the repo command is added to from separate XData blocks.